File: framework/wcf/Basic/Management/ServiceValidation/VB/endpointValidate/EndpointValidateElement.vb
' Copyright (c) Microsoft Corporation.  All Rights Reserved.

Imports System
Imports System.Collections.Generic
Imports System.Configuration
Imports System.ServiceModel
Imports System.ServiceModel.Configuration
Imports System.ServiceModel.Description
Imports System.Xml

Namespace Microsoft.Samples.ServiceModel

    Class EndpointValidateElement
        Inherits BehaviorExtensionElement

        Protected Overloads Overrides Function CreateBehavior() As Object

            Return New EndpointValidateBehavior()

        End Function

        Public Overloads Overrides ReadOnly Property BehaviorType() As Type

            Get

                Return GetType(EndpointValidateBehavior)

            End Get

        End Property

    End Class

End Namespace

File: framework/wcf/Basic/Management/ServiceValidation/CS/endpointValidate/EndpointValidateElement.cs
//----------------------------------------------------------------
// Copyright (c) Microsoft Corporation.  All rights reserved.
//----------------------------------------------------------------

using System;
using System.ServiceModel.Configuration;

namespace Microsoft.Samples.ServiceModel
{
    class EndpointValidateElement : BehaviorExtensionElement
    {
        protected override object CreateBehavior()
        {
            return new EndpointValidateBehavior();
        }

        public override Type BehaviorType
        {
            get { return typeof(EndpointValidateBehavior); }
        }
    }
}
